deploy/manifests/nginx-gateway-experimental.yaml

```diff
@@ -34,6 +34,7 @@ rules:
   - secrets
   - configmaps
   verbs:
+  - get
```

Comment:
hm. I can't see why this needed. In the new code, we're not Getting any of those resources via the API 🤔

Reply:
this is to enable getting namespaces. my previous MR to get `clusterID` was missing this, and leads to error of not able to get `clusterID`.

Reply:
I believe Saylor fixed this with his latest PR: https://github.com/nginxinc/nginx-gateway-fabric/pull/1544.